tests/data/miscellaneous/consecutive_ignore.diff

```diff
@@ -0,0 +1,29 @@
+--- [Deterministic header]
++++ [Deterministic header]
```

Comment:
Instead of a diff test, prefer a test driven by `test_format`; you can put it in the `simple_cases/` directory and separate the input and expected output with a `# output` command. See e.g. `string_prefixes.py` as an example.

Reply:
Gotcha! I updated the test